GLOSSARY.md

```diff
@@ -0,0 +1,40 @@
+# GitOps Glossary
+
+This glossary accompanies the [GitOps Principles](./PRINCIPLES.md), and other supporting documents in this repository.
+
+- ## Continuous
+
+    "Continuous" is intended to match the industry standard term: [reconciliation](#reconciliation) continues to happen, not that it must be instantaneous.
+
+- ## Declarative Description
+
+    A configuration that describes the desired operating state of a system without specifying procedures for how that state will be acheived. This separates configuration (the desired state) from the implementation (commands, API calls, scripts etc.) used to achieve that state.
```

Comment:
```suggestion
    A configuration that describes the desired operating state of a system without specifying procedures for how that state will be achieved. This separates configuration (the desired state) from the implementation (commands, API calls, scripts etc.) used to achieve that state.
```

Reply:
I think we should add Co-authored-by trailers to the commit from #23, since it was recommended here as well. We can still rewrite commits in this PR branch before anything gets committed to `main`.

@todaywasawesome do you want me to take care of that?

Reply:
@scottrigby yes please

Reply:
Done in 2c69564470c4e1bf95ebb2eab1bdb4f25bb37ac8 🙌